mesh_cup:
 normal_data: [0,0,0]

 geometry_type: 0

 object_type: 0

 primitive_type: 0

 overlap: 0

 dimensions: [0,0,0] 

 offset: 0

 weight: 0

